app/models/room.rb

```diff
@@ -37,6 +36,21 @@ def get_setting(name:)
                         .find_by(meeting_option: { name: })
   end
 
+  # Autocreate all meeting options using the default values
+  def create_meeting_options(provider)
+    rooms_configs = MeetingOption.joins(:rooms_configurations)
```

Comment:
I think if we extend `MeetingOption#get_config_value` to fetch single/multiple room configs by name and provider will reduce this boilerplate code and we can leverage it throughout the application.

Reply:
I'll work on this today

Reply:
Done! `get_config_value` returns a hash with the configs values.